web/ui/static/i18n/en_US.json

```diff
@@ -189,6 +189,7 @@
     "label_protocol": "Protocol",
     "label_public_endpoint": "Public Endpoint",
     "label_public_endpoints": "Public Endpoints",
+    "label_ram_threshold": "Auto restart if reached used % of memory",
```

Comment:
Please use the following text:
> Restart when memory usage exceeds % of requested RAM:

Reply:
Value 0 mean: don't restart.
You can find this if statement in file cli/api/daemon.go and line 1111
if getService.RAMThreshold == 0